bin/build-release.sh

```diff
@@ -5,6 +5,7 @@ function _rm {
   rm -rf "$@" 2>/dev/null || true
 }
 RELEASE=`grep -1 -A 0 -B 0 '<version>' pom.xml | head -n 1 | awk '{print $1}' | sed -e 's/.*<version>//' | sed -e 's/<\/version>.*//'`
+SUPER_DOCKERTAG=mesos/storm_supervisor:${RELEASE}.ubuntu1404
```

Comment:
This just creates it in the local docker repo. I was not sure about pushing
an image to any major repo until it this PR was blessed.

On Mon, Jun 1, 2015 at 12:58 PM, Timothy Chen <notifications@github.com>
wrote:

> In bin/build-release.sh
> <https://github.com/mesos/storm/pull/49#discussion_r31445629>:
>
> > @@ -5,6 +5,7 @@ function _rm {
> >    rm -rf "$@" 2>/dev/null || true
> >  }
> >  RELEASE=`grep -1 -A 0 -B 0 '<version>' pom.xml | head -n 1 | awk '{print $1}' | sed -e 's/.*<version>//' | sed -e 's/<\/version>.*//'`
> > +SUPER_DOCKERTAG=mesos/storm_supervisor:${RELEASE}.ubuntu1404
>
> I don't see this image in the mesos docker repo?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesos/storm/pull/49/files#r31445629>.
>